src/components/TodoForm.vue

```diff
@@ -79,7 +82,7 @@ function notCheckedTodos(toDo: Todotype) {
   const index = checkedTodos.value.findIndex((item) => item === toDo);
   if (index !== -1) {
     const uncheckedTodo = checkedTodos.value.splice(index, 1)[0];
-    uncheckedTodo.status = false;
+    uncheckedTodo.isStatus = false;
```

Comment:
Try to find a meaningful name for the variables/attributes. The `isStatus` is not a boolean variable. It should be `isChecked` or `checked`.